### DISCUSSION TITLE
When is 1.0.0 coming out

### DISCUSSION THREAD

**Discussion by mdellagi77** [swift-server/webauthn-swift#98] [Q&A]
Hey. Thanks for this Repo. I see no version updates for over six months. As this Dependency is used in Hummingbird 2 projects I just wanted to know if this is production ready or not?

**Comment by dimitribouniol** [comment]
There is still quite a bit of public enum cleanup to do before we can safely call a 1.0.0, so I would recommend locking to a particular version/commit for now

**Comment by adam-fowler** [comment]
FYI It isn't a dependency of Hummingbird 2. There is an example that uses this package in the hummingbird-examples repo.

**Comment by mdellagi77** [comment]
Thanks for the info and clarifications

**Comment by 0xTim** [comment]
FYI - tracking issues here
